geomstats/geometry/stiefel.py

```diff
@@ -172,6 +144,31 @@ def to_tangent(self, vector, base_point):
         sym_aux = Matrices.to_symmetric(aux)
         return vector - Matrices.mul(base_point, sym_aux)
 
+    def projection(self, point):
+        """Project a close enough matrix to the Stiefel manifold.
+
+        A singular value decomposition is used, and all singular values are
+        set to 1 [Absil]_
+
+        Parameters
+        ----------
+        point : array-like, shape=[..., n, p]
+            Point in embedding manifold.
+
+        Returns
+        -------
+        projected : array-like, shape=[..., n, p]
+
+        References
+        ----------
+        ..[Absil]   Absil, Pierre-Antoine, and Jérôme Malick.
+                    “Projection-like Retractions on Matrix Manifolds.”
+                    SIAM Journal on Optimization 22, no. 1 (January 2012):
+                     135–58. https://doi.org/10.1137/100802529.
+        """
+        u, _, v = gs.linalg.svd(point)
```

Comment:
NIT: I'd rather avoid single letters as variable names (when it's not n, p, etc). For vim users, single variable names are more complicated to find and replace automatically. `mat_u`, `mat_v`?